app/src/main/java/me/ccrama/redditslide/Activities/SettingsSynccit.java

```diff
@@ -49,23 +49,28 @@ public void onCreate(Bundle savedInstanceState) {
             public void onClick(View v) {
                 if (!SettingValues.synccitAuth.isEmpty()) {
 
-                    new AlertDialogWrapper.Builder(SettingsSynccit.this)
-                            .setTitle(R.string.settings_synccit_delete)
-                            .setPositiveButton(R.string.btn_yes, new DialogInterface.OnClickListener() {
-                                @Override
-                                public void onClick(DialogInterface dialog, int which) {
-                                    SettingValues.synccitName = "";
-                                    SettingValues.synccitAuth = "";
-                                    SharedPreferences.Editor e = SettingValues.prefs.edit();
-
-                                    e.putString(SettingValues.SYNCCIT_NAME, SettingValues.synccitName);
-                                    e.putString(SettingValues.SYNCCIT_AUTH, SettingValues.synccitAuth);
-                                    e.apply();
-                                    name.setText(SettingValues.synccitName);
-                                    auth.setText(SettingValues.synccitAuth);
-                                    SynccitRead.visitedIds.removeAll(Collections.singleton("16noez"));
-                                }
-                            }).setNegativeButton(R.string.btn_no, null)
+                    new AlertDialogWrapper.Builder(SettingsSynccit.this).setTitle(
+                            R.string.settings_synccit_delete)
+                            .setPositiveButton(R.string.btn_yes,
+                                    new DialogInterface.OnClickListener() {
+                                        @Override
+                                        public void onClick(DialogInterface dialog, int which) {
+                                            SettingValues.synccitName = "";
+                                            SettingValues.synccitAuth = "";
+                                            SharedPreferences.Editor e = SettingValues.prefs.edit();
+
+                                            e.putString(SettingValues.SYNCCIT_NAME,
+                                                    SettingValues.synccitName);
+                                            e.putString(SettingValues.SYNCCIT_AUTH,
+                                                    SettingValues.synccitAuth);
+                                            e.apply();
+                                            name.setText(SettingValues.synccitName);
+                                            auth.setText(SettingValues.synccitAuth);
+                                            SynccitRead.visitedIds.removeAll(
+                                                    Collections.singleton("16noez"));
```

Comment:
I feel like I did this and the diff is bad but if I did not, in cases where we only added a single item to a collection, it was translated to a `Collections.singleton()` call which is more performant